<launch>

  <arg name="config"                          default="~/.ros/orb-drone.ini"/>>

  <arg name="logo_dir"                        default="~/.ros/aerobot_logo"/>>

  <node pkg="object_detector" name="aerobot_logo_detector_down" type="aerobot_logo_detector.py" output="screen"> </node>

  <node pkg="object_detector" name="qr_detector_down" type="qr_detector.py" output="screen"> </node>

  <node pkg="object_detector" name="qr_detector_front" type="qr_detector.py" output="screen"> </node>

  <node pkg="object_detector" name="aruco_detector_down" type="aruco_detector.py" output="screen"> </node>

  <node pkg="object_detector" name="aruco_detector_front" type="aruco_detector.py" output="screen"> </node>

  <node pkg="object_detector" name="gate_detector_front" type="gate_detector.py" output="screen"> </node>


  <node name="find_object_2d_0" pkg="find_object_2d" type="find_object_2d" output="log">
    <remap from="image"                       to="/object_detector/rgb/image_raw"/>
    <param name="gui"                         value="false" type="bool"/>
    <param name="objects_path"                value="$(arg logo_dir)" type="str"/>         
    <param name="config_file"                 value="$(arg config)" />
  </node>

</launch>

